src/transformers/training_args.py

```diff
@@ -267,6 +267,15 @@ class TrainingArguments:
         eval_delay (`float`, *optional*):
             Number of epochs or steps to wait for before the first evaluation can be performed, depending on the
             eval_strategy.
+        torch_empty_cache_steps (`int`, *optional*):
+            Number of steps to wait before calling `torch.cuda.empty_cache()`. If left unset or set to None, cache will not be emptied.
+
+            <tip>
```

Comment:
```suggestion
            <Tip>
```

Capitol T here if I'm not mistaken :)

Reply:
fixed